releasedir/fs_release_dir.go

```diff
@@ -263,8 +263,14 @@ func (d FSReleaseDir) collectDependentPackages(pkg *boshpkg.Package, allInterest
 	}
 }
 
-func (d FSReleaseDir) writeVendoredPackage(pkg *boshpkg.Package) error {
-	name := pkg.Name()
+func (d FSReleaseDir) writeVendoredPackage(pkg *boshpkg.Package, prefix string) error {
+	var name string
+	if prefix != "" {
+		name = fmt.Sprintf("%s-%s", prefix, pkg.Name())
+	} else {
+		name = pkg.Name()
+	}
+	// name := pkg.Name()
```

Comment:
Should this commented code be removed?

Reply:
nice catch :) will remove it

Reply:
Would you be up for squashing that commit? Would make for cleaner git history.